turbopack/crates/turbopack-core/src/module_graph/merged_modules.rs

```diff
@@ -485,10 +495,7 @@ pub async fn compute_merged_modules(module_graph: Vc<ModuleGraph>) -> Result<Vc<
                     lists.push(after_list.clone());
                     for (i, &m) in after_list.iter().enumerate() {
                         let Some(occurrences) = lists_reverse_indices.get_mut(&m) else {
-                            bail!(
-                                "Couldn't find module in list {:?}",
```

Comment:
merge resolution mistake? keeping the ident here seems potentially useful?

Reply:
1. I've never seen this error once in the last few months
2. This being async messed up `span.entered()`